unittests/test_schedulers.py

```diff
@@ -103,11 +103,6 @@ def minimal_job(make_job):
 @pytest.fixture
 def fake_job(make_job):
     ret = make_job(
-        sched_nodelist='nid000[00-17]',
-        sched_exclude_nodelist='nid00016',
-        sched_partition='foo',
-        sched_reservation='bar',
-        sched_account='spam',
```

Comment:
Would it make sense to pass all the scheduler variables as job options and keep the same expected slurm output?

Reply:
I wouldn't say so. You should instead check that we have at least a test that passes options to job and not just a single one, but multiple, in order to check the order.